#!/usr/bin/python3


def raise_exception():
    return 1 + "m"
